compiler/rustc_error_codes/src/error_codes/E0632.md

```diff
@@ -0,0 +1,29 @@
+An explicit generic argument was provided when calling a function that
+uses `impl Trait` in argument position.
+
+Erroneous code example:
+
+```compile_fail,E0632
+fn foo<T: Copy>(a: T, b: impl Clone) {}
+
+fn main() {
```

Comment:
Nit: you don't need to wrap into a `main` function here.

Reply:
Fixed; thanks for the review.